### DISCUSSION TITLE
Code review IM-11/12/13

### DISCUSSION THREAD

**Discussion by LikerFeed** [HowProgrammingWorks/LiveQA#176] [Code review]
Мы занимаемся разработкой социальной сети с функционалом регистрации/логина, профилями _(аватар, статусы, работа, скиллы)_, постами _(лайки и ответы на посты в разработке....)_, чатами _(в разработке...)_, отображением других юзеров.

В проекте используются React,  Node.JS, PosgreSQL, http module, WebSockets
- Серверная часть - Node.JS, http module(готово на 60%).
- Регистрация/логин - React, Node.JS, PosgreSQL(готово на 100%)
- Профиль - React(готово на 80%)
- Чаты - WebSockets(готово на 10%)
- Юзеры - React, Node.JS, PosgreSQL(готово на 90%)

**Заранее спасибо за ревью!**
[Наш репозиторий](https://github.com/AidXylelele/my-app)

@tshemsedinov

**Comment by tshemsedinov** [comment]
See 1st review https://github.com/AidXylelele/my-app/issues/5
